File: backend/src/main/java/com/parqueando/api/dto/reserva/ReservaResponse.java
package com.parqueando.api.dto.reserva;

import java.math.BigDecimal;
import java.time.LocalDateTime;

public record ReservaResponse(
  Long idReserva,
  String codigoReserva,
  Long idParqueadero,
  Long idVehiculo,
  Long idConductor,
  LocalDateTime fechaHoraInicio,
  LocalDateTime fechaHoraFin,
  BigDecimal totalEstimado,
  String estado,
  String urlComprobante,
  LocalDateTime fechaCreacion
) {}

File: frontend/src/app/models/reserva-response.ts
export interface ReservaResponse {
  idReserva: number;
  codigoReserva: string;
  idParqueadero: number;
  idVehiculo: number;
  idConductor: number;
  fechaHoraInicio: string; // viene ISO con Z
  fechaHoraFin: string;    // viene ISO con Z
  totalEstimado: number;
  estado: 'CONFIRMADA' | 'PENDIENTE' | string;
  urlComprobante?: string;
  fechaCreacion: string;
}